test/modules/standard/IO/saru/getParentName/parentName.good

```diff
@@ -0,0 +1 @@
+/Users/sam/Desktop/chapel/test/modules/standard/IO/saru/getParentName
```

Comment:
Since you're generating this file, you don't want to commit it to the repository.  In addition to removing the file from this PR, you'll want to add it to either the $CHPL_HOME/.gitignore or the .gitignore in the directory for the test